### PR TITLE
f-fix/gh-818-allowance-manual-comment

### LINKED ISSUE #818: Employee allowance manual adjust add comment field

### PR DESCRIPTION
**Pull request by ffpetrovic**
Closes #818

### CODE REVIEW

File: src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js [hunk 1]
```diff
@@ -406,14 +396,36 @@ class ManualAdjust extends Component {
...
                           value={training + values.trainingManualAdjust}
                         />
                       </div>
                     </div>
+                    <div className="d-flex align-items-center pb-2">
+                      <div className="flex-1 w-100" width="10px">
+                        Comment
+                      </div>
+                      <div className="flex-1 w-100 pr-2" />
+                      <div className="flex-1 w-100 pl-2" width="10px">
```
Comment (aberonja) [inline src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js]: Make it full width

File: src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js [hunk 2]
```diff
@@ -406,14 +396,36 @@ class ManualAdjust extends Component {
...
+                        <DrawerInput
```
Comment (aberonja) [inline src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js]: Textarea?

File: src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js [hunk 3]
```diff
@@ -88,11 +90,11 @@ class ManualAdjust extends Component {
     } = this.props.allowance;
 
     return (
-      <div className="d-flex flex-column flex-grow-1">
-        <div className="d-flex justify-content-center pt-4">
+      <div className="row flex-column flex-grow-1">
+        <div className="row justify-content-center pt-4">
```
Comment (aberonja) [inline src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js]: You can't have row in row directly, it'll create horizontal overflow.

File: src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js [hunk 4]
```diff
@@ -139,60 +130,58 @@ class ManualAdjust extends Component {
...
                           className="p-2 w-100 border rounded"
                           disabled
                           value="N/A"
                         />
                       </div>
                     </div>
-                    <div className="d-table-row">
-                      <div className="d-table-cell pb-4" width="10px">
+                    <div className="row pb-2">
+                      <div className="col w-100" width="10px">
```
Comment (aberonja) [inline src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js]: Try it without w-100, I think it'll be fine

Comment (ffpetrovic) [reply src/main/frontend/src/pages/administration/components/manual-adjust/ManualAdjust.js]: I wish... The columns don't end up being exactly the same width.

**Pull request merged**